### PR TITLE
delete onstudy param removed

### PR DESCRIPTION
**Pull request by 650ericy**

### CODE REVIEW

File: src/components/inspectCollection/Actions.tsx [hunk 1]
```diff
@@ -1,18 +1,17 @@
 import React, { useState } from 'react';
 import './Actions.css';
 import { useNavigate } from 'react-router-dom';
-import { exportCollection } from './utils/exportFile.ts';
```
Comment (stephencapper) [inline src/components/inspectCollection/Actions.tsx]: Can this be put back in?

**Review by Battletortoise — APPROVED** [review]
sorry, ignore this

File: src/components/inspectCollection/Actions.tsx [hunk 2]
```diff
@@ -45,15 +44,6 @@ const Actions: React.FC<ActionsProps> = ({ selectedDeck, onStudy }) => {
     }
   };
 
-  // const handleImportDeck = () => {
-  //   console.log('Import Deck clicked');
-  // };
-
-  const handleExportDeck = () => {
-    console.log('Export Deck clicked');
-    exportCollection(selectedDeck.id);
```
Comment (stephencapper) [inline src/components/inspectCollection/Actions.tsx:54]: Put this function back in

File: src/components/inspectCollection/Actions.tsx [hunk 3]
```diff
@@ -62,12 +52,10 @@ const Actions: React.FC<ActionsProps> = ({ selectedDeck, onStudy }) => {
       {showMoreActions && (
         <div className={`additional-actions ${showMoreActions ? 'show' : ''}`}>
           <button onClick={handleDeleteDeck} className="action-button">Delete Deck</button>
-          {/* <button onClick={handleImportDeck} className="action-button">Import Deck</button> */}
-          <button onClick={handleExportDeck} className="action-button">Export Deck</button>
```
Comment (stephencapper) [inline src/components/inspectCollection/Actions.tsx:66]: Put this back back in

File: src/components/inspectCollection/DeckCollection.tsx [hunk 4]
```diff
@@ -20,7 +20,6 @@ type DeckCollectionProps = {
 const DeckCollection: React.FC<DeckCollectionProps> = ({ uid, changeDeck }) => {
   const [decks, setDecks] = useState<DeckType[]>([]);
   const [selectedDeck, setSelectedDeck] = useState<DeckType | null>(null);
-  const [refreshDecks, setRefreshDecks] = useState<boolean>(false);
```
Comment (stephencapper) [inline src/components/inspectCollection/DeckCollection.tsx]: Put this back in

File: src/components/inspectCollection/DeckCollection.tsx [hunk 5]
```diff
@@ -36,7 +35,7 @@ const DeckCollection: React.FC<DeckCollectionProps> = ({ uid, changeDeck }) => {
     };
 
     fetchDecks();
-  }, [uid, refreshDecks]);
```
Comment (stephencapper) [inline src/components/inspectCollection/DeckCollection.tsx]: Put this back in

File: src/components/inspectCollection/DeckCollection.tsx [hunk 6]
```diff
@@ -51,22 +50,15 @@ const DeckCollection: React.FC<DeckCollectionProps> = ({ uid, changeDeck }) => {
 
   return (
     <div>
-      <DeckSelector decks={decks} onDeckSelect={handleDeckSelect} uid={uid} setRefreshDecks={setRefreshDecks} refreshDecks={refreshDecks}/>
```
Comment (stephencapper) [inline src/components/inspectCollection/DeckCollection.tsx:54]: Put these parameters back in

**Review by stephencapper — APPROVED** [review]
Looks good thank you for making the changes!

**Pull request merged**